skglm/penalties/separable.py

```diff
@@ -466,3 +466,48 @@ def is_penalized(self, n_features):
     def generalized_support(self, w):
         """Return a mask with non-zero coefficients."""
         return w != 0
+
+
+class NNLS(BasePenalty):
```

Comment:
LS here means least squares, but we have a penalty, independent of datafit.

Should the name be "PositiveConstraint" ?